src/classes/BGE_BatchGiftEntry_UTIL_TEST.cls

```diff
@@ -86,4 +86,55 @@ private with sharing class BGE_BatchGiftEntry_UTIL_TEST {
         }
     }
 
+    @isTest
+    static void shouldIncludeWidgetFields() {
+        FORM_Element widgetElement =
+                new FORM_Element(
+                        'widget', '', '', '',
+                        new String[]{
+                                UTIL_Namespace.alignSchemaNSWithEnvironment(
+                                        'Account_2_Country'),
```

Comment:
@psadie-sfdo Hey Prince, let me know if this looks right to you.  I can't remember if the field mappings get namespace pre-pended in namespaced scratches, but if so, let me know if you think this looks correct.

Reply:
@kenlewis-sfdo - No namespacing needed for the field mapping developer names. The metadata types and their fields themselves get namespaced, but not the value inside fields like `DeveloperName` in this case.

Reply:
Great - thank you!!